packages/react/src/components/Tooltip/__tests__/DefinitionTooltip-test.js

```diff
@@ -11,11 +11,31 @@ import React from 'react';
 import { DefinitionTooltip } from '../DefinitionTooltip';
 
 describe('DefinitionTooltip', () => {
-  it('should display onClick a definition provided via prop', async () => {
+  it('should display on click a definition provided via prop', async () => {
     const definition = 'Uniform Resource Locator';
     render(<DefinitionTooltip definition={definition}>URL</DefinitionTooltip>);
+    expect(screen.getByRole('button')).toHaveAttribute(
+      'aria-expanded',
+      'false'
+    );
     await userEvent.click(screen.getByText('URL'));
-    expect(screen.getByText(definition)).toBeVisible();
+    expect(screen.getByRole('button')).toHaveAttribute('aria-expanded', 'true');
```

Comment:
wondering if we can keep the check `toBeVisible` along with checking if `aria-expanded` is true

Reply:
Yeah I've seen this before too, the `aria-expanded` does seem like the best workaround 👍